e2e/next-sandbox/test/comments/index.test.ts

```diff
@@ -0,0 +1,42 @@
+import type { Page } from "@playwright/test";
+import { test } from "@playwright/test";
+
+import {
+  genRoomId,
+  getJson,
+  preparePages,
+  waitForJson,
+  waitUntilEqualOnAllPages,
+} from "../utils";
+
+const TEST_URL = "http://localhost:3007/comments";
+
+test.describe("Threads", () => {
+  let pages: [Page, Page];
+
+  test.beforeEach(async ({}, testInfo) => {
+    const room = genRoomId(testInfo);
+    pages = await preparePages(`${TEST_URL}?room=${encodeURIComponent(room)}`);
+  });
+
+  test.afterEach(() => Promise.all(pages.map((page) => page.close())));
+
+  test("verify A and B display same number of threads after threads are loaded", async () => {
+    await waitForJson(pages, "#isLoading", false);
+    await waitUntilEqualOnAllPages(pages, "#numOfThreads");
+  });
+
+  test("verify thread creation on B is broadcasted correctly to A", async () => {
+    const [page1, page2] = pages;
```

Comment:
This tests for the broadcasting of thread/comment creation. We use event broadcasting to also relay information about thread metadata update, comment update, and comment deletion, so this test provides validity to those actions too. That being said, it might still be helpful to have a test item for each of the actions.